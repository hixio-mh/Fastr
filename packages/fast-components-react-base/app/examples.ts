import AutoSuggestExamples from "../src/auto-suggest/examples.data";
export { AutoSuggestExamples };

import BadgeExamples from "../src/badge/examples.data";
export { BadgeExamples };

import BreadcrumbExamples from "../src/breadcrumb/examples.data";
export { BreadcrumbExamples };

import ButtonExamples from "../src/button/examples.data";
export { ButtonExamples };

import CardExamples from "../src/card/examples.data";
export { CardExamples };

import CheckboxExamples from "../src/checkbox/examples.data";
export { CheckboxExamples };

import ContextMenu from "../src/context-menu/examples.data";
export { ContextMenu };

import ContextMenuItem from "../src/context-menu-item/examples.data";
export { ContextMenuItem };

import DialogExamples from "../src/dialog/examples.data";
export { DialogExamples };

import DividerExamples from "../src/divider/examples.data";
export { DividerExamples };

import HorizontalOverflowExamples from "../src/horizontal-overflow/examples.data";
export { HorizontalOverflowExamples };

import HypertextExamples from "../src/hypertext/examples.data";
export { HypertextExamples };

import ImageExamples from "../src/image/examples.data";
export { ImageExamples };

import LabelExamples from "../src/label/examples.data";
export { LabelExamples };

import ListboxExamples from "../src/listbox/examples.data";
export { ListboxExamples };

import ListboxItemExamples from "../src/listbox-item/examples.data";
export { ListboxItemExamples };

import NumberFieldExamples from "../src/number-field/examples.data";
export { NumberFieldExamples };

import ProgressExamples from "../src/progress/examples.data";
export { ProgressExamples };

import RadioExamples from "../src/radio/examples.data";
export { RadioExamples };

import SelectExamples from "../src/select/examples.data";
export { SelectExamples };

import SliderExamples from "../src/slider/examples.data";
export { SliderExamples };

import SliderTrackItemExamples from "../src/slider-track-item/examples.data";
export { SliderTrackItemExamples };

import TabsExamples from "../src/tabs/examples.data";
export { TabsExamples };

import TextAreaExamples from "../src/text-area/examples.data";
export { TextAreaExamples };

import TextFieldExamples from "../src/text-field/examples.data";
export { TextFieldExamples };

import ToggleExamples from "../src/toggle/examples.data";
export { ToggleExamples };

import TypographyExamples from "../src/typography/examples.data";
export { TypographyExamples };
