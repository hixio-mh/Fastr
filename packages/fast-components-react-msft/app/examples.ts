import ButtonExamples from "../src/button/examples.data";
export { ButtonExamples };

import CallToActionExamples from "../src/call-to-action/examples.data";
export { CallToActionExamples };

import CaptionExamples from "../src/caption/examples.data";
export { CaptionExamples };

import CheckboxExamples from "../src/checkbox/examples.data";
export { CheckboxExamples };

import Dialog from "../src/dialog/examples.data";
export { Dialog };

import DividerExample from "../src/divider/examples.data";
export { DividerExample };

import FlipperExample from "../src/flipper/examples.data";
export { FlipperExample };

import HeadingExample from "../src/heading/examples.data";
export { HeadingExample };

import HypertextExamples from "../src/hypertext/examples.data";
export { HypertextExamples };

import ImageExamples from "../src/image/examples.data";
export { ImageExamples };

import LabelExamples from "../src/label/examples.data";
export { LabelExamples };

import MetatextExamples from "../src/metatext/examples.data";
export { MetatextExamples };

import ParagraphExamples from "../src/paragraph/examples.data";
export { ParagraphExamples };

import ProgressExamples from "../src/progress/examples.data";
export { ProgressExamples };

import SubheadingExamples from "../src/subheading/examples.data";
export { SubheadingExamples };

import ToggleExamples from "../src/toggle/examples.data";
export { ToggleExamples };

import TextFieldExamples from "../src/text-field/examples.data";
export { TextFieldExamples };

import TypographyExamples from "../src/typography/examples.data";
export { TypographyExamples };
