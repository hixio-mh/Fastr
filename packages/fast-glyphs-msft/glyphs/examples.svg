<svg width="16" height="16" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="Canvas">
<g id="Group">
<g id="Vector">
<path d="M 4 2.9L 5.6 3.8L 5 4.8L 3.4 3.9L 3.4 5.8L 2.3 5.8L 2.3 3.9L 0.6 4.8L -3.8147e-07 3.8L 1.7 2.9L 0.1 2L 0.7 1L 2.2 1.9L 2.2 2.38419e-08L 3.3 2.38419e-08L 3.3 1.9L 4.9 1L 5.5 2L 4 2.9Z" transform="translate(10.3999 1.09998)"/>
</g>
<g id="Vector_2">
<path d="M 4 2.9L 5.6 3.8L 5 4.8L 3.4 3.9L 3.4 5.8L 2.2 5.8L 2.2 3.9L 0.6 4.8L 4.76837e-08 3.8L 1.6 2.9L 4.76837e-08 2L 0.6 1L 2.2 1.9L 2.2 -1.90735e-07L 3.3 -1.90735e-07L 3.3 1.9L 4.9 1L 5.5 2L 4 2.9Z" transform="translate(1.19995 10.2)"/>
</g>
<g id="Vector_3">
<path d="M 0 8L 0 0L 8 0L 8 8L 0 8ZM 1 1L 1 7L 7 7L 7 1L 1 1ZM 14.4 13.1L 16 14L 15.4 15L 13.8 14.1L 13.8 16L 12.7 16L 12.7 14.1L 11 15.1L 10.4 14.1L 12 13.2L 10.4 12.3L 11 11.3L 12.6 12.2L 12.6 10.3L 13.7 10.3L 13.7 12.2L 15.3 11.3L 15.9 12.3L 14.4 13.1Z"/>
</g>
</g>
</g>
</svg>