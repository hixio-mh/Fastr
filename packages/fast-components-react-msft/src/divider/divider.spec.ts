import examples from "./examples.data";
import {
    generateSnapshots,
    SnapshotTestSuite,
} from "@microsoft/fast-jest-snapshots-react";
import { DividerProps } from "./index";

describe("divider snapshots", (): void => {
    generateSnapshots(examples as SnapshotTestSuite<DividerProps>);
});
