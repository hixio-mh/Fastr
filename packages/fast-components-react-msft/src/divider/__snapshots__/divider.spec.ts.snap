// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`divider snapshots Divider: 0 1`] = `
<hr
  className="divider-0-1-1"
  data-sketch-symbol="Divider"
/>
`;

exports[`divider snapshots Divider: 1 1`] = `
<hr
  className="divider-0-1-2"
  role="presentation"
/>
`;

exports[`divider snapshots Divider: 2 1`] = `
<hr
  className="divider-0-1-3"
/>
`;
